example/src/APIClient.js

```diff
@@ -1,20 +1,38 @@
-import { environment } from './Configuration';
+import { environment, channel } from './Configuration';
+
+const serverConfiguration = {
+  channel: channel,
+  shopperReference: 'Checkout Shopper',
+  reference: 'React Native'
+};
+
+const parseConfig = (configuration) => {
+    return {
+      merchantAccount: configuration.merchantAccount,
+      countryCode: configuration.countryCode,
+      shopperLocale: configuration.shopperLocale,
+      amount: configuration.amount
+  }
+};
```

Comment:
You can use an implicit return in this case

```suggestion
const parseConfig = (configuration) => ({
    merchantAccount: configuration.merchantAccount,
    countryCode: configuration.countryCode,
    shopperLocale: configuration.shopperLocale,
    amount: configuration.amount
});
```

And given that the properties are called the same, you can use shorthand property names
```suggestion
const parseConfig = ({ merchantAccount, countryCode, shopperLocale, amount }) => ({
    merchantAccount,
    countryCode,
    shopperLocale,
    amount
});
```